altos-rust/port/cortex-m0/src/peripheral/gpio/mod.rs

```diff
@@ -95,7 +91,7 @@ impl GPIO {
         }
     }
 
-    /// Enable a GPIO group, you must do this before you can set any
+    /// Enable a GPIO group. You must do this before you can set any
```

Comment:
Don't really care for the word "You" here. Maybe something like: "This must be done before setting any pins within a group."